include/oneapi/tbb/detail/_machine.h

```diff
@@ -55,6 +60,10 @@ static inline void yield() {
     int err = sched_yield();
     __TBB_ASSERT_EX(err == 0, "sched_yiled has failed");
 }
+#elif __TBBMALLOC_BUILD && _WIN32
+static inline void yield() {
```

Comment:
```suggestion
// Use Windows API for yield in tbbmalloc to avoid dependency on C++ runtime with some implementations.
static inline void yield() {
```